packages/controller-utils/jest.config.js

```diff
@@ -18,8 +18,8 @@ module.exports = merge(baseConfig, {
     global: {
       branches: 68.05,
       functions: 80.55,
-      lines: 69.82,
-      statements: 70.17,
+      lines: 69.01,
```

Comment:
I'm dropping the coverage thresholds here because all I added was data. I am not sure whether there is a way to tell the coverage tool to skip these kinds of changes.

Reply:
Hmm. should we skip them? They're still part of the package's public API.

Maybe we can snapshot them or something.

Reply:
Fair, right, we don't actually want to skip these lines. I think this might be a deficiency in how we are running tests. Currently we run them per package, so Jest only sees one package at a time. It doesn't know where this package is being referenced across the monorepo.

Reply:
Right, there is that aspect too. Depends on our goal I guess; having direct package-level coverage by unit tests sounds reasonable to me, even if they are covered elsewhere. The package is the API we want to safeguard.

Reply:
Oh, definitely, I just meant that this is why the coverage reporter is failing to notice that the variable I added is indeed being used. And this sort of thing is a problem for all of our packages at the moment. So even if we wanted to raise the thresholds across the board, we wouldn't be able to do that.

You mentioned snapshotting — can you go into more detail about that?

Reply:
Sure. If we want to ensure something exists in an API but there is no functionality we'd want a unit test for,  we can use a snapshot. i.e. the Jest snapshot API.

We can write a unit test that creates a snapshot from the set of all constants. That ensures that the constants can't change without us noticing, letting us refactor them and move them around without worrying about how it might break the public API.